config/dev.exs

```diff
@@ -46,7 +46,10 @@ config :guardian, Guardian,
   secret_key: "e62fb6e2746f6b1bf8b5b735ba816c2eae1d5d76e64f18f3fc647e308b0c159e"
 
 config :code_corps, :analytics, CodeCorps.Analytics.InMemoryAPI
+
+# Configures stripe for dev mode
 config :code_corps, :stripe, Stripe
+config :code_corps, :stripe_env, :dev
```

Comment:
`Mix.env` is not available in production, so if we want to differentiate between livemode events on different environments, we need to set an explicit config value. 

Right now, in the code below, the events are setup so that production will ignore `livemode: false` events and other environments will ignore `livemode: true` events.